### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#308] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181351](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/625ebd6e-3ab6-40fd-a6f9-d381e2f5fc72)

**Comment by IonutCicio** [comment]
Si consideri il gruppo $(G, \star)$ e $H, H' \leq G$, verificare che $H_1\cap H_2 = H_I\leq G$. Usando il criterio, basta verificare che $\forall h_1, h_2 \in H_I, h_1 \star h_2^{-1} \in H_I$. 

#### L'intersezione di 2 sottogruppi è un sottogruppo

Presi $h_1, h_2 \in H_I$, si ha che $h_1, h_2$ sono sia in $H_1$ sia in $H_2$ _(essendo nell'intersezione),_ ma, per il punto 2. della definizione di sottogruppo, si ha che $h_2^{-1} \in H_1$ e che $h_2^{-1} \in H_2$, quindi $h_2^{-1} \in H_I$ _(dato che sta in entrambi i sottogruppi, sta anche nell'intersezione)._ Usando il punto 1. della definizione di sottogruppo, si ha che $h_1 \star h_2^{-1} \in H_1$ e che $h_1 \star h_2^{-1} \in H_2$, per cui $h_1 \star h_2^{-1} \in H_I$

#### L'intersezione di $n$ sottogruppi è un sottogruppo

Si procede per induzione sul numero di sottogruppi. Il **caso base** $n = 2$ è stato già dimostrato al punto precedente, quindi si può procedere con il **passo induttivo**.

Sapendo che $\bigcap\limits_{i = 1}^{n} H_i \leq G$ bisogna dimostrare che $\bigcap\limits_{i = 1}^{n + 1} H_i \leq G$ è un sottogruppo, sapendo che $\forall i = 1, 2, ..., n + 1, H_i \leq G$. Si può riscrivere $\bigcap\limits_{i = 1}^{n + 1} H_i$ come $H_L = \bigcap\limits_{i = 1}^{n} H_i, H_R= H_{n + 1}, H_I = H_L \cap H_R$, per cui la dimostrazione è simile al passo base.

Presi $h_1, h_2 \in H_I$, si ha che $h_1, h_2$ sono sia in $H_L$ sia in $H_R$ _(essendo nell'intersezione),_ ma, per il punto 2. della definizione di sottogruppo, si ha che $h_2^{-1} \in H_L$ e che $h_2^{-1} \in H_R$, quindi $h_2^{-1} \in H_I$ _(dato che sta in entrambi i sottogruppi, sta anche nell'intersezione)._ Usando il punto 1. della definizione di sottogruppo, si ha che $h_1 \star h_2^{-1} \in H_L$ e che $h_1 \star h_2^{-1} \in H_R$, per cui $h_1 \star h_2^{-1} \in H_I$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/7954769f-770d-4d7d-9aa3-ebf2cf299090)
**Soluzione alternativa**
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/beb5f742-5765-488c-b082-1f2bf3b910cc)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-16 190715](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/5c40fd81-ff1d-4d8b-9d21-1769a65f4b2f)
